recipes-kernel/linux/linux-mainline-common.inc

```diff
@@ -17,6 +17,3 @@ KBUILD_DEFCONFIG:qemuriscv64 = "defconfig"
 KBUILD_DEFCONFIG:freedom-u540 = "defconfig"
 
 COMPATIBLE_MACHINE = "(qemuriscv32|qemuriscv64|freedom-u540)"
-
-KERNEL_FEATURES:remove = "features/debug/printk.scc"
-KERNEL_FEATURES:remove = "features/kernel-sample/kernel-sample.scc"
```

Comment:
This removes it for everything, this isn't the right way

Reply:
Thanks for the feedback, @alistair23  I see your point. I’ll adjust it to target only the specific configurations where removing these features is necessary.